### PR TITLE
Add bigint-to-uint256-hint

### PR DESCRIPTION
**Pull request by Juan-M-V**
# bigint_to_uint256

## Description

Add Implementation for bigint to uint 256

## Checklist
- [x] Integration tests added.

### CODE REVIEW

File: src/vm/hints/secp/bigint_utils.rs [hunk 1]
```diff
@@ -45,17 +55,34 @@ pub fn nondet_bigint3(
...
+// %{ ids.low = (ids.x.d0 + ids.x.d1 * ids.BASE) & ((1 << 128) - 1) %}
+pub fn bigint_to_uint256(
+    vm: &mut VirtualMachine,
+    ids: &HashMap<String, BigInt>,
+    hint_ap_tracking: Option<&ApTracking>,
+) -> Result<(), VirtualMachineError> {
+    let x_struct = get_relocatable_from_var_name("x", ids, vm, hint_ap_tracking)?;
+    let d0 = vm.memory.get_integer(&x_struct)?;
+    let d1 = vm.memory.get_integer(&(&x_struct + 1))?;
+    let low = (d0 + d1 * &*BASE_86) & ((bigint!(1) << 128_usize) - 1_usize);
```
Comment (Oppen) [inline src/vm/hints/secp/bigint_utils.rs:69]:
```suggestion
    let low = (d0 + d1 * &*BASE_86) & bigint_u128(u128::MAX);
```

File: src/vm/hints/secp/secp_utils.rs [hunk 2]
```diff
@@ -13,8 +18,7 @@ pub fn split(integer: &BigInt) -> Result<[BigInt; 3], VirtualMachineError> {
     if integer.is_negative() {
         return Err(VirtualMachineError::SecpSplitNegative(integer.clone()));
     }
-    let base = bigint!(1) << 86_usize;
-    let base_max = base - bigint!(1);
+    let base_max = &*BASE_86 - bigint!(1);
```
Comment (Oppen) [inline src/vm/hints/secp/secp_utils.rs:21]: This can also be `lazy_static!`.

**Review by Oppen — CHANGES_REQUESTED** [review]

File: src/vm/hints/secp/secp_utils.rs (hunk 2 above)
Comment (Juan-M-V) [reply src/vm/hints/secp/secp_utils.rs]: resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/329/commits/ea093c1dac6342877c95a5c6dc001593608f795f

File: src/vm/hints/secp/bigint_utils.rs (hunk 1 above)
Comment (Juan-M-V) [reply src/vm/hints/secp/bigint_utils.rs]: Resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/329/commits/ea093c1dac6342877c95a5c6dc001593608f795f

**Review by Oppen — APPROVED** [review]

**Pull request merged**